src/widgets/ot_pulltorefresh.eliom

```diff
@@ -63,24 +66,26 @@ module Make (Conf : CONF) = struct
 
   let touchmove_handler ev _ =
     scroll_handler ();
-    Dom_html.stopPropagation ev;
-    if !dragStart >= 0
-    then
-      if !refreshFlag
-      then Dom.preventDefault ev
-      else if ev##.touches##.length = 1
-      then (
-        let target = ev##.changedTouches##item 0 in
-        Js.Optdef.iter target (fun target ->
-            distance :=
-              Float.sqrt (float_of_int (- !dragStart + target##.clientY))
-              *. scale);
-        (*move the container if and only if at the top of the document and
+    if%lwt Lwt.return (not !scrollingX)
```

Comment:
Why if%lwt? (not really neccessary and makes code more difficult to read)

Reply:
I thought it was better than having a `else Lwt.return_unit` but I don't need that either, turns out. I misread the parenthesis stack.